services/ui-src/src/components/statusing/StatusIcon.tsx

```diff
@@ -0,0 +1,54 @@
+import { Flex, Image, Text } from "@chakra-ui/react";
+import { ReportType } from "types";
+// assets
+import successIcon from "assets/icons/icon_check_circle.png";
+import errorIcon from "assets/icons/icon_error_circle_bright.png";
+import { assertExhaustive } from "utils/other/typing";
+
+export const StatusIcon = ({
+  reportType,
+  status,
+}: {
+  reportType: ReportType;
+  status?: boolean;
+}) => {
+  switch (reportType) {
+    case ReportType.SAR:
+    case ReportType.WP: {
+      if (status) {
+        return (
+          <Flex sx={sx.status}>
+            <Image src={successIcon} alt="Success notification" />
+            <Text>Complete</Text>
+          </Flex>
+        );
+      } else if (status === undefined) {
+        return <></>;
+      } else {
+        return (
```

Comment:
When is this scenario for? If `status` is `null`?

Reply:
for the Error Icon? It's if status is false

Reply:
I see. I'd probably refactor this to a switch statement instead to make it clearer:
```
switch (status) {
  case true:
    return (
      <Flex sx={sx.status}>
        <Image src={successIcon} alt="Success notification" />
        <Text>Complete</Text>
      </Flex>
    );
  case false:
    return (
      <Flex sx={sx.status}>
        <Image src={errorIcon} alt="Error notification" />
        <Text>Error</Text>
      </Flex>
    );
  default:
    return <></>;
}
```